scripts/deploy/govcms-enable-modules

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+#
+# GovCMS enable required modules
+#
+# The GovCMS platform provides some modules to assist with running on
+# the platform, this will ensure they're enabled every deploy.
+#
+
+LAGOON_ENVIRONMENT_TYPE=${LAGOON_ENVIRONMENT_TYPE:-production}
+GOVCMS_DEPLOY_ENABLE_MODULES=${GOVCMS_DEPLOY_ENABLE_MODULES:-true}
+
+if [ "$GOVCMS_DEPLOY_ENABLE_MODULES" != "true" ]; then
+  echo "[Skip]: Enable default modules for GovCMS"
+  exit 0
+fi
+
+echo "GovCMS Deploy :: Enable modules"
+
+MODULES=$(drush pm:list --status=enabled)
+NON_PROD_MODULES=("stage_file_proxy")
+PLATFORM_MODULES=(
+  "redis"
+  "fast404"
+  "clamav"
+  "robotstxt"
+  "lagoon_logs"
+  "environment_indicator"
+)
+
+if [ "$LAGOON_ENVIRONMENT_TYPE" != "production" ]; then
+  for MODULE in "${NON_PROD_MODULES[@]}"; do
```

Comment:
do we really not want the PLATFORM_MODULES enabled in non-prod?

Reply:
This was just for parity with govcms_lagoon (it's enabled on both I believe - will double check).

Reply:
FWIW, my preferred is to leave it installed (avoid install/uninstalling Drupal modules as a general rule) and do this for production sites in *.settings.php
```
      $config['stage_file_proxy.settings']['origin'] = NULL;
```

Reply:
While I agree in principle - there are some modules that we need to ensure are _always_ enabled for security and policy reasons. If configuration management is up to a site to manage then they will be able to disable these modules.

I think this might warrant a bigger discussion about how we want to mandate platform requirements.